src/components/homePage.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<link rel="stylesheet" href="../styles/root.css" />
+	</head>
+	<body>
+		<div
+			class="InstallationDomMainContainer TypoHtag card-container-bg flex-space-evenly"
+		>
+			<div
+				class="padding-small-card shade classA translateOne"
+			
+			>
+			<a 	href="./RulesOne.html" target="iframe-quiz">
+				<div
+					class="quiz-container padding-small-card shade imgTransition"
+				>
+					<img
+						class="quiz-img"
+						src="./img/quiz1.jpg"
+						alt="quiz1-Img"
+						srcset=""
+					/>
+				</div>
+				<div class="padding-normal card-selector-text">
+					<h4>Lorem Ipsum</h4>
+					<p>lorem Ipsum is simply dummy text</p>
+				</div>
+				</a>
+			</div>
+
+			<div
+				class="padding-small-card shade classA translateOne"
+			>
```

Comment:
suggestion - use descriptive class names as shade & classA don't give any context.

Reply:
Okay, I'll do that.

Reply:
I used that class in my CL Project I imported some class properties from it. I will change it. 

Thank you Aniket for Detail PR Review.